juriscraper/opinions/united_states/state/ill.py

```diff
@@ -6,8 +6,6 @@
    2014-12-02: Updated by Mike Lissner to remove the summaries code.
    2016-02-26: Updated by arderyp: simplified thanks to new id attribute identifying decisions table
    2016-03-27: Updated by arderyp: fixed to handled non-standard formatting
-   2021-11-02: Status validation based on Amendment to Rule 23 rulings
-               https://chicagocouncil.org/illinois-supreme-court-amendment-to-rule-23/
 """
```

Comment:
Also -  you should follow the pattern we have above for date updated - by whom and what was changed.

Reply:
Added comment with change to both scrapers.